config.ts

```diff
@@ -60,6 +61,7 @@ const websiteConfig = z.strictObject({
   discord: z.string(),
   instagram: z.string(),
   linkedin: z.string(),
+  hackathon: z.string(),
```

Comment:
can we make this a bit more descriptive. Or actually wanna just inline the url directly to the code and make this a `show_hackathon_modal` or something. The code is too specific to be configurable we just want to be able to turn it off.